### PR TITLE
Fix 500 Error from /session-expired and /signed-out

### PR DESCRIPTION
**Pull request by shahi645**
### What is the context of this PR?
feature includes:

- when signed in user gets session-expired/signed-out with theme and title of survey
launched

- when not signed in (and no cookie) theme is set as default and no title

Resolves: #1230

Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Load survey runner launch a survey and navigate to 'localhost:5000/session-expired' or 'localhost:5000/signed-out'. page should match survey and survey title. 

Load survey, delet cookie, then navigate to above pages....user should see default page and no title

### CODE REVIEW

File: app/helpers/session_helper.py [hunk 1]
```diff
@@ -19,5 +19,4 @@ def end_session_with_schema_context(schema=None):
 
     _remove_survey_session_data()
 
-    session['theme'] = schema['theme']
-    session['survey_title'] = schema['title']
+
```
Comment (ajmaddaford) [inline app/helpers/session_helper.py]: Extra line should be removed

**Review by ajmaddaford — APPROVED** [review]
The commit message should be `Fix 500 error from /session-expired`

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
I think we should have a test for the `/session-expired` and `/signed-out` urls returning a 200 rather than a 500.

**Review by collisdigital — APPROVED** [review]
Tested locally; works as described.

**Review by collisdigital — CHANGES_REQUESTED** [review]
Changes to be made as discussed so that theme is used even on 401 if possible

**Review by collisdigital — APPROVED** [review]
Checked with @charlotte-hirst and working as needed; please squash.

File: app/views/errors.py [hunk 2]
```diff
@@ -94,3 +95,13 @@ def get_tx_id():
...
+
+
+def render_template(template_name):
+    tx_id = get_tx_id()
+    user_agent = user_agent_parser.Parse(request.headers.get('User-Agent', ''))
+
+    return render_theme_template(session.get('theme', 'default'), template_name=template_name,
+                                 analytics_ua_id=current_app.config['EQ_UA_ID'],
+                                 ua=user_agent, tx_id=tx_id,
+                                 survey_title=TemplateRenderer.safe_content(session.get('survey_title', '')))
```
Comment (ajmaddaford) [inline app/views/errors.py]: Each argument should be on a new line as it's far easier to read

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by yoganandkunche — APPROVED** [review]
@shahi645 Tested locally.. happy with the solution and it works as expected.

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**